### ISSUE TITLE
Intel 5300 driver

### ISSUE THREAD

**Issue by zhiyueyan** [dhalperi/linux-80211n-csitool-supplementary#177]
Where could I find Intel 5300 driver ? Any address ? Thanks a lot.

**Comment by dpward** [comment]
The (regular) driver for this wireless adapter is part of the [Linux kernel](https://www.kernel.org) and is called _[iwlwifi](https://wireless.wiki.kernel.org/en/users/drivers/iwlwifi)_. If you are using any popular Linux distribution for PCs, such as [Ubuntu](http://www.ubuntu.com/desktop), the _iwlwifi_ driver ships with it and will be loaded automatically whenever you boot the system if the wireless adapter is installed.

Apart from the driver itself, a firmware image needed to use the wireless adapter is made available under a different license from Intel. The firmware is also shipped with these same Linux distributions and will automatically be loaded by the _iwlwifi_ driver into the adapter.

So in other words, the wireless adapter should (normally) work out-of-the-box as soon as you boot your system.

However, to use the CSI Tool, you need to make use of both a modified _iwlwifi_ driver and a modified firmware image. See the [installation instructions](http://dhalperi.github.io/linux-80211n-csitool/installation.html) to get started.

If you are specifically looking to browse the source code to the modified driver, it is in the [git repository referenced in the instructions](https://github.com/dhalperi/linux-80211n-csitool/tree/master/drivers/net/wireless/iwlwifi/).
